proto/cache.proto

```diff
@@ -30,11 +36,96 @@ message CacheStats {
   int64 total_cached_action_exec_usec = 11;
 }
 
+// Request to retrieve detailed per-request cache stats.
+message GetCacheScoreCardRequest {
+  context.RequestContext request_context = 1;
+
+  // The invocation ID for which to look up cache stats.
+  string invocation_id = 2;
+
+  // Pagination token. Not intended for direct client usage.
+  message PageToken {
+    // The start index in the scorecard results list, after sorting and
+    // filtering results. Number of results to fetch is determined by the
+    // server.
+    int64 index = 1;
+  }
+
+  // A string representation of the page token, which must be a value returned
+  // by a previous response.
+  string page_token = 3;
+
+  // TODO(bduffany): add sorting/filtering options
+}
+
+message GetCacheScoreCardResponse {
+  context.ResponseContext response_context = 1;
+
+  // The cache results for the current page.
+  repeated ScoreCard.Result results = 2;
+
+  // A token used to fetch more results. If empty, there are no results
+  // remaining.
+  string page_token = 3;
```

Comment:
nit: call this next_page_token (to allow us to add prev_page_token later if we want)

Reply:
Done